src/openai/_client.py

```diff
@@ -3,11 +3,13 @@
 from __future__ import annotations
 
 import os
-from typing import TYPE_CHECKING, Any, Union, Mapping
+from typing import TYPE_CHECKING, Any, Union, Mapping, Callable, Awaitable
 from typing_extensions import Self, override
 
 import httpx
 
+from openai._models import FinalRequestOptions
```

Comment:
tiny nit: 
```suggestion
from ._models import FinalRequestOptions
```
(sorry we don't have lint rules for this yet)